src/models/k8sresource.ts

```diff
@@ -52,6 +55,9 @@ interface K8sResource {
   };
   /** result of schema validation */
   validation?: ResourceValidation;
+
+  /** result of policy validation */
+  issues?: ResourceValidation;
```

Comment:
I'd like to refactor validation to maybe be more elaborate, I kept it split off now as validateResource overwrites the whole object and I do not validatePolicy of all resources on editing file for performance.

Reply:
Resolving this to be tackled later.